docs/software/sciapps/namd.md

```diff
@@ -1,4 +1,274 @@
+            --with-tcl --tcl-prefix ${DEV_VIEW_PATH} \
+            --with-fftw --with-fftw3 --fftw-prefix ${DEV_VIEW_PATH} \
+            --cuda-gencode arch=compute_90,code=sm_90 --with-single-node-cuda --with-cuda --cuda-prefix ${DEV_VIEW_PATH}
+        cd Linux-ARM64-g++.cuda && make -j 32
+        
+        # The namd3 executable (GPU-accelerated) will be built in the Linux-ARM64-g++.cuda directory
+        ```
+
+        * Change `<PATH_TO_NAMD_SOURCE>` to the path where you have the NAMD source code
+        * Change `<NAMD_UENV>` to the name (or path) of the actual NAMD UENV you want to use
+
+        To run NAMD, make sure you load the same UENV and view you used to build NAMD, and set the following variable:
+
+        ```bash
+        export LD_LIBRARY_PATH="${DEV_VIEW_PATH}/lib/"
+        ```
+
+    === "CPU Build"
+
+        Some workflows, such as constant pH MD simulations, might require a CPU-only NAMD build which is used to drive the simulation.
+
+        !!! warning "Use the CPU-only build only if needed"
+            The CPU-only build is optional and should be used only if needed. You should use it in conjunction with the GPU build to drive the simulation.
+            Do not use the CPU-only build for actual simulations as it will be slower than the GPU build.
+
+        You can build a CPU-only version of NAMD as follows:
+        
+        ```bash
+        export DEV_VIEW_NAME="develop-single-node"
+        export PATH_TO_NAMD_SOURCE=<PATH_TO_NAMD_SOURCE>
+
+        # Start uenv and load develop view
+        uenv start --view=${DEV_VIEW_NAME} <NAMD_UENV>
+
+        # Set variable VIEW_PATH to the view
+        export DEV_VIEW_PATH=/user-environment/env/${DEV_VIEW_NAME}
+
+        cd ${PATH_TO_NAMD_SOURCE}
+        ```
+
+        !!! info "Action required"
+            Modify the `<PATH_TO_NAMD_SOURCE>/arch/Linux-ARM64.tcl` file now.
+            Change `-ltcl8.5` with `-ltcl8.6` in the definition of the `TCLLIB` variable.
+        
+        ```bash
+        # Build bundled Charm++
+        tar -xvf charm-8.0.0.tar && cd charm-8.0.0
+        ./build charm++ multicore-linux-arm8 gcc --with-production --enable-tracing -j 32
+
+        # Configure NAMD build for GPU
+        cd ..
+        ./config Linux-ARM64-g++ \
+            --charm-arch multicore-linux-arm8-gcc --charm-base $PWD/charm-8.0.0 \
+            --with-tcl --tcl-prefix ${DEV_VIEW_PATH} \
+            --with-fftw --with-fftw3 --fftw-prefix ${DEV_VIEW_PATH}
+        cd Linux-ARM64-g++ && make -j 32
+
+        # The namd3 executable (CPU-only) will be built in the Linux-ARM64-g++ directory
+        ```
+
+        * Change `<PATH_TO_NAMD_SOURCE>` to the path where you have the NAMD source code
+
+        To run NAMD, make sure you load the same UENV and view you used to build NAMD, and set the following variable:
+
+        ```bash
+        export LD_LIBRARY_PATH="${DEV_VIEW_PATH}/lib/"
+        ```
+
+=== "Multi-node build"
```

Comment:
The content in the Single-node build and Multi-node build tabs is quite long, so it is hard to see where the tabbed content ends.
These might be better off as `## sections`?
If they are sections, then they also show up in the navigation list on the right hand side.

Reply:
Good point. This was a deliberate decision to hide the multi-node build, since it does not support GPU-resident mode which is a major improvement compared to the GPU-offload mode. The average system can't scale over multiple nodes, therefore the single-node build is almost always the one to use. The tabs are essentially the whole page (until the useful links).

That said, I'm happy to change it to sections if you think it is better/clearer. I will change the note on the multi-node build to a warning to steer the users to the single-node build.